finagle-redis/src/main/scala/com/twitter/finagle/redis/SentinelClient.scala

```diff
@@ -0,0 +1,85 @@
+package com.twitter.finagle.redis
+
+import com.twitter.finagle.builder.ClientBuilder
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.{ Service, ServiceFactory }
+import com.twitter.util.Future
+import org.jboss.netty.buffer.{ ChannelBuffers, ChannelBuffer }
+
+object SentinelClient {
+
+  /**
+   * Construct a sentinel client from a single host.
+   * @param host a String of host:port combination.
+   */
+  def apply(host: String): SentinelClient = SentinelClient(
+    ClientBuilder()
```

Comment:
Is it ok with the following piece of code?

    def apply(host: String): SentinelClient = {
      val client = com.twitter.finagle.Redis.client
      SentinelClient(
        client
          .configured(client.params[DefaultPool.Param].copy(high = 1))
          // Daemonize is private to the builder package.
          // Is there another way to set this? Or can we just ignore it?
          // .configured(client.params[Daemonize](true))
          .newService(host))
    }